fenix/detekt-baseline.xml

```diff
@@ -568,6 +569,7 @@
     <ID>UndocumentedPublicFunction:DefaultSyncInteractor.kt$SyncInteractor$fun onCameraPermissionsNeeded()</ID>
     <ID>UndocumentedPublicFunction:DefaultToolbarMenu.kt$DefaultToolbarMenu$@VisibleForTesting(otherwise = PRIVATE) fun canAddToHomescreen(): Boolean</ID>
     <ID>UndocumentedPublicFunction:DefaultToolbarMenu.kt$DefaultToolbarMenu$@VisibleForTesting(otherwise = PRIVATE) fun canInstall(): Boolean</ID>
+    <ID>UndocumentedPublicFunction:DefaultToolbarMenu.kt$DefaultToolbarMenu$@VisibleForTesting(otherwise = PRIVATE) fun shouldShowOpenInRegularTab(): Boolean</ID>
```

Comment:
Let's just add a kDoc for `shouldShowOpenInRegularTab` to avoid this

Reply:
I thought it would be quite odd for that being the only function in that file with a kDoc, but OK!

Reply:
done!